### PR TITLE
Resolving all linting issues.

### PR DESCRIPTION
**Pull request by caleb-mabry**
This is a massive PR that needs to be tested thoroughly since we don't have any unit tests that really exist right now.
The code can be tested here http://ec2-3-221-159-94.compute-1.amazonaws.com/

### CODE REVIEW

**Comment by caleb-mabry** [comment]
Pull request is currently under user review!

**Comment by stonedDiscord** [comment]
that is one massive merge conflict that seems to come from 2 different linters?????

File: webAO/client/setEmote.js [hunk 1]
```diff
@@ -1,43 +1,44 @@
+/* eslint no-restricted-syntax: 'off' */
 import fileExistsSync from '../utils/fileExistsSync';
-
+import transparentPNG from '../constants/transparentPNG';
```
Comment (stonedDiscord) [inline webAO/client/setEmote.js:3]: 😂

File: webAO/utils/fileExists.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
-const fileExists = async (url) => new Promise((resolve, reject) => {
```
Comment (stonedDiscord) [inline webAO/utils/fileExists.js:1]: sus

**Review by stonedDiscord — COMMENTED** [review]
haven't looked at client and master yet, rest looks good

**Comment by caleb-mabry** [comment]
You may be wondering about the new indents. This was brought in with the change of the `.eslint` config file. Defining the rule as just tabs and seeing it there makes it easier to implement/view for others

File: webAO/master.js [hunk 3]
```diff
@@ -1,151 +1,138 @@
 import FingerprintJS from '@fingerprintjs/fingerprintjs';
-
-import { unescapeChat, safeTags } from './encoding.js';
-
-const myStorage = window.localStorage;
```
Comment (stonedDiscord) [inline webAO/master.js:5]: did you delete the localStorage

**Review by stonedDiscord — COMMENTED** [review]
this pretty much renames every variable in the game

File: webAO/master.js (hunk 3 above)
Comment (caleb-mabry) [reply webAO/master.js:5]: Yes, I deleted `myStorage` because you didn't reference the variable anywhere.

**Comment by caleb-mabry** [comment]
> this pretty much renames every variable in the game

Yeah, it really does. Mainly because the naming convention of those variables is python-esque.

**Comment by caleb-mabry** [comment]
This is literally asking to be a nightmare.

**Pull request closed without merging**